manifests/server.pp

```diff
@@ -76,6 +76,7 @@
   Boolean $create_admin                                 = $mongodb::params::create_admin,
   String $admin_username                                = $mongodb::params::admin_username,
   Optional[String] $admin_password                      = undef,
+  String $admin_password_hash                           = undef,
```

Comment:
Shouldn't this be `Optional`? Perhaps this and `admin_password` should both be `Optional[String[1]]`

Reply:
You're right, that should be optional